### DISCUSSION TITLE
Creating a CITE-seq reference for ref mapping

### DISCUSSION THREAD

**Discussion by adamgayoso** [satijalab/seurat#5026] [Q&A]
I'd like to create a reference dataset for reference mapping just like in the Seurat v4 paper (in the case of multiple CITE-seq samples). Is there any code to reproduce how this dataset (`pbmc_multimodal.h5seurat`) was generated?

https://satijalab.org/seurat/articles/multimodal_reference_mapping.html#a-multimodal-pbmc-reference-dataset

**Comment by adamgayoso** [comment]
It would be great if there was some code to reproduce this. In particular we are also confused about the correct margin for CLR. In the methods of the Seurat v4 manuscript it is stated that

```
We apply a CLR transformation to normalize protein data within each cell.
```

which by my interpretation would be `margin=1`. However, the WNN tutorial shows `margin=2`.

**Reply by rsatija** [reply]
Thanks! In the methods when we said 'Within a cell', we meant that each cell (column) is normalized independently, so thats why we use margin = 2.

We're putting together a vignette that shows an example of integrating multiple CITE-seq datasets and then performing WNN, and will respond to this comment when we post.

**Reply by adamgayoso** [reply]
Thanks!
